src/lib/components/ui/ArticleCard.svelte

```diff
@@ -3,11 +3,15 @@
 	import { slide } from 'svelte/transition';
 	import Badge from './badge/badge.svelte';
 
-	const { article }: { article: ArticleMetadata } = $props();
+	const TRANSITION_DURATION = 300 as const;
```

Comment:
```suggestion
	const TRANSITION_DURATION = 300;
```

You don't need `as const` here, as it's not an object or array. `const` is already enough to make the value immutable